server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/SetMessagesMethodStepdefs.java

```diff
@@ -135,4 +138,18 @@ public void setFlags(String message, List<String> keywords) throws Throwable {
             .discardContent();
         mainStepdefs.awaitMethod.run();
     }
+
+    @When("^the user set flags (.*) via messageIdProbe of \"([^\"]*)\" on \"([^\"]*)\" of user \"([^\"]*)\"")
```

Comment:
By the way this is JMAP aren't we supposed to speak about keywords and not about flag?

Reply:
But we use probe, not JMAP call so I think flags are still make sence